caiman/utils/utils.py

```diff
@@ -137,15 +139,15 @@ def si_parse(imd):
 
     """
 
-    imd = imd.split('\n')
-    imd = [i for i in imd if '=' in i]
-    imd = [i.split('=') for i in imd]
-    imd = [[ii.strip(' \r') for ii in i] for i in imd]
-    imd = {i[0]: val_parse(i[1]) for i in imd}
-    return imd
+    imddata:Any = imd.split('\n')
```

Comment:
Using a different variable lets us have the input be restricted to be a string type